backend/controllers/LocationsController.ts

```diff
@@ -160,7 +171,7 @@ const LocationsController = {
 
   async postPersonalLocation(req: Request, res: Response) {
     try {
-      const { locationName, description, coordinates } = req.body;
+      const { locationName, description, coordinates, filters } = req.body;
 
```

Comment:
change type of coordinates from [string,string] to [number, number]

Reply:
done